hawc/apps/common/api/viewsets.py

```diff
@@ -37,6 +37,7 @@ class CleanupFieldsBaseViewSet(
     def get_queryset(self):
         return self.model.objects.all()
 
+    # should we catch the error here?
     @action(detail=False, methods=["get"])
```

Comment:
Is this the right spot to catch the error so it could be used for all 500 errors with the api?

Reply:
I wonder if we just catch this exception here might do the trick? https://github.com/shapiromatron/hawc/blob/ec785ff7d0e6918ba0e6cdf9dd66ca56e8eaf195/hawc/apps/common/api/mixins.py#L55-L59

We already have a response of 400 that may be appropriate; the issue is probably that we're not catching a `django.db.utils.DataError` so it's not being handled

Reply:
if that works, we can write a test for it?